### DISCUSSION TITLE
Navigation within Firefly

### DISCUSSION THREAD

**Discussion by arnoldbird** [iotaledger/firefly#2026] [Feedback]
## Issue description

The navigation UI in the left column of Firefly is confusing.  The icon on the top appears to be something I can click, but nothing happens when I click it.  I get a mouse pointer when I hover over the second icon, but clicking on it doesn't do anything.

When I click on my-first-initial-in-a-disc at the bottom of the column, a small UI opens up that is ambiguous and perhaps unnecessary. When it opened, I tried clicking on my name at the top before figuring out that that is not something I can click on.  It serves no purpose to put my name there.  So that leaves the settings icon and the logout link.  Why not put these directly in the left column of the app instead of making me open a separate UI just to show me the three options... which turn out to be two options.

Why is the navigation distributed the way it is in the left column, with two icons at the top and two at the bottom.  Should be just...

* home
* settings
* logout

If you're going to use the wallet icon for the "home" screen, there needs to be some visual indication that that's where I am when I open the app, so the user understands where they are.  The conventional cue would be something like a arrow or stripe/border to the left of the active link.

Another possibility would be just...

* settings
* logout

...and just have the settings open in an overlay atop the main UI, and then let the user x out that overlay to return to the wallet.

Make the navigation as simple and obvious as possible.

## Wallet version

0.3.1

## System specification

Ubuntu 18.

Paste any errors that you see from the Error Log in the Firefly menu or share screenshots of the issue.

**Comment by arnoldbird** [comment]
Feels a bit more comfortable to use on day 2, for what it's worth.  It doesn't really take long to figure out where things are, and from that point things like the branding in the upper left become less distracting.

**Comment by dev0pa** [comment]
> When I click on my-first-initial-in-a-disc at the bottom of the column, a small UI opens up that is ambiguous and perhaps unnecessary. 

...and why "click" first, why not open the menus on hover?
